pkg/security/secl/model/model.go

```diff
@@ -101,6 +101,7 @@ type NetworkContext struct {
 	Destination      IPPortContext `field:"destination"`       // destination of the network packet
 	NetworkDirection uint32        `field:"network_direction"` // SECLDoc[network_direction] Definition:`Network direction of the network packet` Constants:`Network directions`
 	Size             uint32        `field:"size"`              // SECLDoc[size] Definition:`Size in bytes of the network packet`
+	Type             uint32        `field:"type"`              // SECLDoc[type] Definition:`Type of the network packet` Constants:`Network Protocol Types`
```

Comment:
As it's only related to ICMP packets, should we put is as `omitempty` ?

Reply:
For now yes only for ICMP. omitemty is for JSON marshal things not for SECL. I have put omitempty on the serializer side.